### PR TITLE
Add config flow to `pushbullet`

### PR DESCRIPTION
**Pull request by engrbm87**
## Proposed change

Add config flow to `pushbullet`.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/74240
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/pushbullet.markdown [hunk 1]
```diff
@@ -24,90 +27,35 @@ The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushe
...
 
 Notification Mirroring allows users to see their Android device's notifications on their computer. It must be first enabled in the app and is currently only available on the Android platform. For more information, please see [this announcement](https://blog.pushbullet.com/2013/11/12/real-time-notification-mirroring-from-android-to-your-computer/) on the Pushbullet Blog.
 
 Go to [https://www.pushbullet.com/#settings/account](https://www.pushbullet.com/#settings/account) to retrieve your API key/access token.
 
-### Configuration
+{% include integrations/config_flow.md %}
 
-To enable the Pushbullet sensor in your installation, add the following to your `configuration.yaml` file:
+### Sensors
```
Comment (MartinHjelmare) [inline source/_integrations/pushbullet.markdown]:
```suggestion
## Sensors
```

File: source/_integrations/pushbullet.markdown [hunk 2]
```diff
@@ -24,90 +27,35 @@ The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushe
 
 </div>
 
-### Sensor
-
-The `pushbullet` sensor platform reads messages from [Pushbullet](https://www.pushbullet.com/), a free service to send information between your phones, browsers, and friends. This sensor platform provides sensors that show the properties of the latest received Pushbullet notification mirror.
-
 ### Setup
```
Comment (frenck) [inline source/_integrations/pushbullet.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/pushbullet.markdown [hunk 3]
```diff
@@ -24,90 +27,35 @@ The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushe
...
+- `Application name`: The application sending the push.
+- `Body`: The notification message body.
+- `Notification ID`: The ID of the notification.
+- `Tag`: The notification tag (if the application sending supports it).
+- `Package name`: The name of the sender’s package.
+- `Receiver Email`: The email of the push’s target.
+- `Sender Email`: The email of the sender.
+- `Sender device ID`: The ID of the sender’s device.
+- `Title`: The title of the push.
+- `Type`: The type of the push.
```
Comment (frenck) [inline source/_integrations/pushbullet.markdown]: This is not code and should thus not be code fenced.

```suggestion
- Application name: The application sending the push.
- Body: The notification message body.
- Notification ID: The ID of the notification.
- Tag: The notification tag (if the application sending supports it).
- Package name: The name of the sender’s package.
- Receiver Email: The email of the push’s target.
- Sender Email: The email of the sender.
- Sender device ID: The ID of the sender’s device.
- Title: The title of the push.
- Type: The type of the push.
```

File: source/_integrations/pushbullet.markdown [hunk 4]
```diff
@@ -24,90 +27,35 @@ The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushe
...
-    source_device_iden:
-      description: ID of the sender's device.
-    title:
-      description: Title of the push.
-    type:
-      description: Type of push.
-{% endconfiguration %}
-
-All properties will be displayed as attributes. The properties array are just for logging the sensor readings for multiple properties.
+`Body` and `Title` are enabled by default. The rest can be enabled from the UI.
```
Comment (frenck) [inline source/_integrations/pushbullet.markdown]:
```suggestion
"Body" and "Title" are enabled by default. The rest can be enabled from the UI.
```

File: source/_integrations/pushbullet.markdown [hunk 5]
```diff
@@ -24,90 +27,35 @@ The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushe
...
 
 ## Notifications
 
 The `pushbullet` notification platform sends messages to [Pushbullet](https://www.pushbullet.com/), a free service to send information between your phones, browsers, and friends. The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushes per month.
```
Comment (frenck) [inline source/_integrations/pushbullet.markdown]:
```suggestion
The Pushbullet notification platform sends messages to [Pushbullet](https://www.pushbullet.com/), a free service to send information between your phones, browsers, and friends. The free tier is [limited](https://docs.pushbullet.com/#push-limit) to 500 pushes per month.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @engrbm87 👍

**Pull request merged**